seed.js

```diff
@@ -1,7 +1,7 @@
-const commentDb = require('./db/index');
+const db = require('./db/connect');
 const loremIpsum = require("lorem-ipsum").LoremIpsum;
 
-const maxComments = Math.floor(Math.random() * 200) + 37;
+const maxComments = 20000;
 const maxSongLength = 480; // in seconds, how long song is
 
```

Comment:
maximum comments 20,000, so for 1 million records would this be run 50 times?

Reply:
Sorry, this is an outdated PR. For 1 million records, the maxComments should be set to 1 million.